### ISSUE TITLE
Option To Change Display Of Answer Buttons

### ISSUE THREAD

**Issue by ThePieMonster** [ankidroid/Anki-Android#9476]
Currently there are four answer buttons that are present as seen in the screenshot below. It would be great if a option existed that allowed only the "again" and "good" buttons to be displayed instead of "again, "hard", "good", and "easy".

![Screenshot_20210901-221653](https://user-images.githubusercontent.com/11359134/131776400-5892e886-08f8-4056-a0c2-7c8d816ab24a.png)

**Comment by mikehardy** [comment]
Hi there! This would be a fundamental change to the way Anki as a system works, and we would only do this if you pursued it upstream with AnkiDesktop first and it was accepted there: https://forums.ankiweb.net/

As a possible alternative, you can build a completely different reviewer experience using our Javascript API if you like https://github.com/ankidroid/Anki-Android/wiki/AnkiDroid-Javascript-API - and using that you could present as many (or as few buttons) as you like, mapping them to whatever actual Anki response you like

**Comment by ThePieMonster** [comment]
> Hi there! This would be a fundamental change to the way Anki as a system works, and we would only do this if you pursued it upstream with AnkiDesktop first and it was accepted there: https://forums.ankiweb.net/
> 
> As a possible alternative, you can build a completely different reviewer experience using our Javascript API if you like https://github.com/ankidroid/Anki-Android/wiki/AnkiDroid-Javascript-API - and using that you could present as many (or as few buttons) as you like, mapping them to whatever actual Anki response you like

Thanks for the response, however I disagree that it would be a fundamental change to Anki. All that is being mentioned here is not using the Hard or Easy buttons. All four buttons have a impact on the underlying algorithm, and depending on what you are studying it may be more beneficial to use just the Again or Good buttons. I have been doing this for years now, making sure to only press the "Again" or "Good" buttons during my reviews. However this is not a good method as sometimes another option is pressed and the reviews are messed up as a result. 

In conclusion, I'm just looking to hide 2 out of the 4 review buttons.

**Comment by mikehardy** [comment]
> All that is being mentioned here...

...is changing the number of options built in the system from what exists now to some different state.

As reviewing is fundamental to the system, that's a fundamental change.

I hear your request, but my response and recommended course of action still stands.

**Comment by ThePieMonster** [comment]
> > All that is being mentioned here...
> 
> ...is changing the number of options built in the system from what exists now to some different state.
> 
> As reviewing is fundamental to the system, that's a fundamental change.
> 
> I hear your request, but my response and recommended course of action still stands.


I hear you, but I just want to create a option to hide 2 options, not change the total number of options to the system. 
Think of it as a UI change that can be toggled in settings. 

Is the below what its in control of determining what answer buttons to display? 
https://github.com/ankidroid/Anki-Android/wiki/AnkiDroid-Javascript-API

**Comment by mikehardy** [comment]
You would need to use that API in combination with full screen mode (where normal reviewer buttons and everything are hidden). You would create a javascript snippet in your template that displayed buttons exactly how you want, and have them do exactly what you want.

Adding formal options to the system that change fundamental aspects is not will not happen. If upstream does it, we will follow

**Comment by ThePieMonster** [comment]
Got it working upstream. To reiterate though, although there are only 2 buttons displayed, it's no different than having all 4 answer buttons displayed. Just not going to click 2 out of the 4 so why look at em.

![image](https://user-images.githubusercontent.com/11359134/132081884-f43f71a6-64f5-4335-a697-e3088eb98fbc.png)

**Comment by mikehardy** [comment]
That's fantastic! I'm not against the idea, it is just one of my primary tasks here to maintain ecosystem compatibility - if they accept that for merge as a feature, we can follow here
